bee-protocol/src/workers/heartbeater.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    types::metrics::NodeMetrics,
+    workers::{
+        packets::HeartbeatPacket, peer::PeerManager, sender::Sender, storage::StorageBackend, MetricsWorker,
+        PeerManagerResWorker,
+    },
+};
+
+use bee_network::PeerId;
+use bee_runtime::{node::Node, shutdown_stream::ShutdownStream, worker::Worker};
+use bee_tangle::{Tangle, TangleWorker};
+
+use async_trait::async_trait;
+use futures::stream::StreamExt;
+use log::info;
+use tokio::time::interval;
+use tokio_stream::wrappers::IntervalStream;
+
+use std::{any::TypeId, convert::Infallible, time::Duration};
+
+const _HEARTBEAT_SEND_INTERVAL: Duration = Duration::from_secs(30);
+const _HEARTBEAT_RECEIVE_INTERVAL: Duration = Duration::from_secs(100);
+const CHECK_HEARTBEATS_INTERVAL: Duration = Duration::from_secs(5);
```

Comment:
Maybe add `_SECONDS`?

Reply:
I would say it would be a little bit annoying to have to change all the names in case we change the value. I think `Duration::from_secs` already shows intent enough idk